feat(plugins-doc) update documentation for centreon-plugin-applications-protocol-[cifs,ftp,sftp] plugins (#3269)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-protocol-cifs.md

import TabItem from '@theme/TabItem';

## Pack assets

### Templates

The Monitoring Connector **Protocol CIFS** brings a host template:

* **App-Protocol-Cifs-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="App-Protocol-Cifs-custom" label="App-Protocol-Cifs-custom">

| Service Alias | Service Template                    | Service Description                      |
|:--------------|:------------------------------------|:-----------------------------------------|
| Connection    | App-Protocol-Cifs-Connection-custom | Check connection on a remote CIFS server |

> The services listed above are created automatically when the **App-Protocol-Cifs-custom** host template is used.

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias | Service Template                     | Service Description                  |
|:--------------|:-------------------------------------|:-------------------------------------|
| Files-Count   | App-Protocol-Cifs-Files-Count-custom | Check number of files in a directory |

> | Files-Date    | App-Protocol-Cifs-Files-Date-custom  | Check modified time of files         |

Reviewer: ```suggestion
| Files-Date    | App-Protocol-Cifs-Files-Date-custom  | Check last modified time of files         |
```

> To monitor your CIFS server, you can use username authentication.

Reviewer: ```suggestion
To monitor your CIFS server, the plugin needs to log in to it using a username and a password.
```

| EXTRAOPTIONS       | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options). | --verbose              |             |

</TabItem>
</Tabs>

3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
is able to monitor a resource using a command like this one (replace the sample values by yours):

```bash
/usr/lib/centreon/plugins/centreon_protocol_cifs.pl \
	--plugin=apps::protocols::cifs::plugin \
	--mode=scenario \
	--hostname='10.0.0.1' \

> 	--cifs-username='' \
> 	--cifs-password='' \

Reviewer: ```suggestion
	--cifs-username='my-username' \
        --cifs-password='my-password' \
```
